src/V3Ast.h

```diff
@@ -1719,7 +1719,7 @@ class AstNode VL_NOT_FINAL {
     // Data type locators
     AstNodeDType* findBitDType() { return findBasicDType(VBasicDTypeKwd::LOGIC); }
     AstNodeDType* findDoubleDType() { return findBasicDType(VBasicDTypeKwd::DOUBLE); }
-    AstNodeDType* findStringDType() { return findBasicDType(VBasicDTypeKwd::STRING); }
+    AstNodeDType* findStringDType() const { return findBasicDType(VBasicDTypeKwd::STRING); }
```

Comment:
If this one is const the neighbors presumably can be too, please submit a pull request to fix these I'll merge first.